src/common/viz/FigureExtractor.js

```diff
@@ -194,5 +216,14 @@ define(['./Utils'], function (Utils) {
         return this._metaNodesMap[metaTypeId];
     };
 
+    const extractPointsArray = function (pair) {
+        const pointsArr = pair.split(',').map(num => parseFloat(num));
+        let cartesianPoints = {x: pointsArr[0], y: pointsArr[1]};
```

Comment:
minor point but `cartesionPoints` appears to refer to a single point. Maybe it should be called `cartesionPoint`?